test/parallel/test-fs-exists.js

```diff
@@ -54,3 +54,12 @@ assert(!fs.existsSync(`${f}-NO`));
 assert(!fs.existsSync());
 assert(!fs.existsSync({}));
 assert(!fs.existsSync(new URL('https://foo')));
+
+{
+  // This test is to ensure that the v8 fast api works.
+  const oneBytePath = 'hello.txt';
+  for (let i = 0; i < 1e5; i++) {
+    assert(!fs.existsSync(oneBytePath));
```

Comment:
This should be moved to `test/pummel` instead. But also, in general we need to avoid running tight loops in the tests to avoid introducing timeouts in the CI on the slower machines. Maybe it's already enough that the fast path is exercised in the benchmark..

Reply:
Can't we use V8 natives API like they do unit test fast calls? IIRC you need to `%PrepareForOptimization(fn)`, call the function, `%OptimizeOnNextCall(fn)`, and call it again. That last call should be optimized.

Reply:
Also, while I'm not good enough with C++ to suggest how to implement it, I think we can put something into place (maybe only in debug builds?). For example, the fast version, when called, increases some counter that we can get from JavaScript for an assertion.

Reply:
I think we can keep an array of booleans for all fast APIs to see if they are called, and expose them to the JS land, toggling a boolean shouldn't be very expensive.

Reply:
For this PR though doing  `%PrepareForOptimization()` and `%OptimizeOnNextCall()` in the tests may be fine. But we also need to check in JS land if the optimizing compiler is enabled at all in the test to avoid failing on builds that turns optimizations off, which would be tricky..